integration/entrypoint.sh

```diff
@@ -117,13 +117,21 @@ function validate_mnt_number {
     fi
 }
 
+function set_config_option {
+    KEY="${1}"
+    VALUE="${2}"
+
+    sed -i "s/\($KEY *= *\).*/\1$VALUE/" "${SNAPSHOTTER_CONFIG}"
```

Comment:
This works but not generic since it ignores the toml hierarchy. It's fine for now